packages/react-server/core/ClientController.js

```diff
@@ -532,6 +538,9 @@ class ClientController extends EventEmitter {
 	 * DOM node(s) that were created.
 	 */
 	_cleanupPreviousRender(index) {
+
+		index || (index = 0); // Default to everything.
```

Comment:
Nit: can we use `index = index || 0` here? I think it's a little easier to understand the intent. (And it's actually two characters shorter ;))